protoc_plugin/test/protos/google/protobuf/unittest_nested_any.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package protobuf_unittest_nested_any;
+
+option csharp_namespace = "Google.Protobuf.TestProtos";
+option java_multiple_files = true;
+option java_package = "com.google.protobuf.test";
```

Comment:
Could you remove these lines? They're not used in this test.

Reply:
Okay, lines removed and rebased onto master.